mappings/net/minecraft/util/random/RandomGenerator.mapping

```diff
@@ -8,6 +8,17 @@ CLASS net/minecraft/unmapped/C_rlomrsco net/minecraft/util/random/RandomGenerato
 		COMMENT @throws IllegalArgumentException if {@code max} isn't positive
 		ARG 1 max
 			COMMENT the upper limit on values (exclusive)
+	METHOD m_aokkzmee nextTriangular (DD)D
+		COMMENT {@return a random {@code double} between {@code mode - deviation} and
+		COMMENT {@code mode + deviation} (both inclusive) with mode {@code mode}}
```

Comment:
```suggestion
		COMMENT {@code mode + deviation} (both exclusive) with mode {@code mode}
```
`}`, and pretty sure these are exclusive, unless `nextDouble()` can return `1.0`

Reply:
the extra bracket is closing `{@return`

Reply:
oops, didn't notice the `@return` had brackets

but for `nextDouble`:
<https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Random.html#nextDouble()>
> The general contract of nextDouble is that one double value, chosen (approximately) uniformly from the range 0.0d (inclusive) to 1.0d (**exclusive**), is pseudorandomly generated and returned.

Reply:
got it!
```suggestion
		COMMENT {@code mode + deviation} (both exclusive) with mode {@code mode}}
```